inspirehep/utils/bibtex.py

```diff
@@ -318,7 +318,7 @@ def _get_collaboration(self):
         collaboration = ''
         if 'collaboration' in self.record:
             try:
-                collaboration = self.record['collaboration'][0]
+                collaboration = self.record['collaboration'][0]['value']
             except IndexError:
```

Comment:
No, that would have to be an added step. It will either return an empty list of no values exist or a list of values.